### ISSUE TITLE
Potion Melder duplicates Potions

### ISSUE THREAD

**Issue by SeooY** [EnigmaticaModpacks/Enigmatica9#750]
### Modpack Version

1.15.1

### Describe your issue.

With the current configuration, where only 1 dose of each input potion is consumed to create 2 doses of the output potion, the Potion Melder can duplicate potions. This can happen by melding the intended duplication target with either just water, a potion with no effects (i.e. Awkward Potion), or a potion with an effect already existing in the target potion.

https://streamable.com/5pxomu

This allows duplicating extremely complicated potions for free (with water), or with very basic ingredients like a single Nether Wart.

### Have you modified the modpack?

No

### Did the issue happen in singleplayer or on a server?

Singleplayer

### Discord Username

seoy

**Comment by MuteTiefling** [comment]
I'm inclined to let this stay for a few reasons:

1. Fixing it would mean a heavy nerf to most people's power generation capabilities
2. Setting it up to dupe is arguably as much or more effort than just creating the potions normally.
3. Ultimately, you're just saving on some easily farmed plants since all potions have been given easy farmable ingredients. 

It does feel weird, however, that non effect potions can be melded at all. So I'll relay that to Ars Nouveau, at least.

**Comment by MuteTiefling** [comment]
Relayed: https://discord.com/channels/743298050222587978/1171186670813589564

**Comment by SeooY** [comment]
I agree, I don't think the Potion Melder output should be reduced to default. From my point of view it ideally should be fixed directly in Ars Nouveau, as the mod itself provides the option to increase the Melder output in the config, which causes this issue when the input potions aren't validated. 

If it was me I would also check that the input potions don't have duplicated effects (so you can't just keep adding the same potion over and over), to balance it further, but I understand that check is more complex and removing non-effect potions will already alleviate most of the issue.
